ui/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js

```diff
@@ -301,7 +303,7 @@ export default class ImportWithSeedPhrase extends PureComponent {
             this.handleConfirmPasswordChange(event.target.value)
           }
           error={confirmPasswordError}
-          autoComplete="confirm-password"
+          autoComplete="new-password"
```

Comment:
according to https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete confirming a new password should use the `new-password` value